spec/numberformat.html

```diff
@@ -30,9 +30,19 @@ <h1>SetNumberFormatDigitOptions ( _intlObj_, _options_, _mnfdDefault_, _mxfdDefa
           1. Set _intlObj_.[[MaximumSignificantDigits]] to _mxsd_.
         1. Else if _mnfd_ is not *undefined* or _mxfd_ is not *undefined*, then
           1. Set _intlObj_.[[RoundingType]] to ~fractionDigits~.
-          1. Let _mnfd_ be ? DefaultNumberOption(_mnfd_, 0, 20, _mnfdDefault_).
+          1. If _mxfd_ is *undefined*, then
+            1. Let _mnfdActualDefault_ be _mnfdDefault_.
+          1. Else,
+            1. Set _mxfd_ to ? ToNumber(_mxfd_).
+            1. If _mxfd_ is *NaN*, then throw a *RangeError* exception.
```

Comment:
Nit: No "then" in single-line steps.

Reply:
@leobalter do you feel strongly about this?

Reply:
Omitting "then" in single-line steps is the normal convention in ECMA-262 and ECMA-402.

Reply:
@anba is correct